node/index.d.ts

```diff
@@ -1076,6 +1076,40 @@ declare module "zlib" {
     export function unzip(buf: Buffer, callback: (error: Error, result: Buffer) => void): void;
     export function unzipSync(buf: Buffer, options?: ZlibOptions): Buffer;
 
+    export var constants: {
```

Comment:
May be you should make this namespace and the actual constants like Z_NO_FLUSH as export var declarations